src/main/java/org/jenkinsci/plugins/workinghours/WorkingHoursConfig.java

```diff
@@ -131,4 +136,24 @@ public void setExcludedDates(
         this.excludedDates = value;
         save();
     }
+
+    /**
+     * Gets the message to be displayed when the job is blocked
+     *
+     * @return default message if the value is null
+     */
+    public String getCustomJobHoldText() {
```

Comment:
I think we should rename this to make the code easier to read for people not familiar - when I read this it sounds like it only handles the custom text, but that's only 1/2 of what it does. 

I actually think `getJobHoldText` would be clearer, but I'm open to suggestions

Reply:
Yeah, `getJobHeldText` sounds much better :+1: